data/human/news.txt

```diff
@@ -2150,3 +2150,313 @@ news "graffiti on spaceport wall"
+	word
+		"There's "
+	word
+		"a lone pirate prodding the ship with rudimentary tools. "
+		"a few pirates watching but not apparently touching it. "
+		"noone othere, the area around your ship is almost suspiciously devioid of people. "
+	word
+		"Whatever's going on your ship appears unscathed."
+
+news "Microgravity"
+	location
+		planet Mordente-Bridi
+name
+	"Repairs"
+message
+	word
+		"Watching the micro-gravity section of the spaceport from the bar, you "
+	word
+		"observe "
+		"notice "
+		"spot "
+		"spy "
+	word
+		"a Bastion "
+		"an Aerie "
+		"a Falcon "
+		"a Leviathan "
+	word
+		"returning from a raid, "
+	word
+		"potmarked with laser burns but no apparent serious damage. "
+		"battered with thousands of small perferations. Somebody's turned a mainstay of the pirate arsenal, gatling guns, back on them. "
+		"with substantial sections of hull heavily battered or missing altogether. Fair guess they didn't have enough anti-missle. "
+	word
+		"As it comes to a stop in the maitenance bay engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"The work won't take the shuttles long."
+		"They'll be working for the rest of the shift on this one."
+
+news "Symbol War"
+	location
+		planet Gagarin
+name
+	"Guerrilla Symbology"
+message
+	word
+		"It's an average day in a busy spaceport, "
+	word
+		"there's no real quiet anywhere in the port, "
+		"it's the expected amout of noisy crowd background noise, "
+		"with the normal buzz of activity with thousands of things happening every second, "
+	word
+		"it can be a bit much but anyone who works with ships learns to tune it out. "
+	word
+		"In spite of the drive to tune out the rumble of the crowds "
+		"However, against the drive to tune out the crowds "
+		"A rare and fleeting drop in the rumble allows% "
+	word
+		"you hear something that stands out "
+	word
+		"turning your attention to the unexpected noise, "
+		"glancing over to the new commotion, "
+	word
+		"you see a squad of "
+	word
+		"CCOR irregulars "
+		"CCOR guerrillas "
+	word
+		"a few of them are armed but most have paint, plaster, and ladders."
+	word
+		"They "
+	word
+		"leave the spaceport traffic alone,"
+		"aren't disrupting traffic,"
+	word
+		"rather "
+		"instead "
+	word
+		"they are "
+	word
+		"climbing the back wall toward the giant crest of the planetary government% "
+		"walking along the peripheral walls of the port focusing on the small government crests "
+		"returning to CCOR crests that have been covered with planetary government crests "
+	word
+		"they dilligently replace the government crests with CCOR's. "
+	word
+		"Spaceport security are "
+	word
+		"noticably absent."
+		"actively ignoring them."
+		"keeping their distance."
+	replace
+		"% you" " you to"
+		"% they dilligently replace the government crests" " they dilligently replace the large crest"
+		"%" ""
+
+news "IP Theft"
```

Comment:
This one also feels long and awkwardly phrased for spaceport news.

Reply:
It probably is. I wanted to put something together right when inspiration struck.
Could probably cut the current beginning down and restructure it a bit, I'll think about how to do so.

Reply:
Rewrote it a bit. I think something may be wonky, it wasn't showing your suggested changes for this one until I went to edit it.